common/app/layout/Sublink.scala

```diff
@@ -303,5 +307,51 @@ case class ContentCard(
     case _ => false
   }
 }
+object ContentCard {
 
+  def makeFromApiContent(apiContent: contentapi.Content): Option[ContentCard] = {
+
+    apiContent.fields.map { fields =>
+      val discussionSettings = for {
+        commentable <- fields.commentable
+        isClosedForComments <- fields.commentCloseDate.map(closeDate => new DateTime(closeDate.dateTime).isBeforeNow)
+      } yield DiscussionSettings(commentable, isClosedForComments, None)
+
+      val isVideo = apiContent.`type` == ContentType.Video
+      val faciaCardHeader = FaciaCardHeader(false, false, isVideo, false, false, None, apiContent.webTitle, EditionalisedLink(apiContent.webUrl))
+      val unsetDisplaySettings = DisplaySettings(false, false, false, false, false)
+      val cardTypesForRecommendations = ItemClasses(mobile = MediaList, tablet = Standard, None)
+
+      def byline: Option[Byline] = {
+        val contributorApiTags = apiContent.tags.filter(tag => tag.`type` == TagType.Contributor)
+        val contributorTags = contributorApiTags.map(tag => model.Tag(TagProperties.make(tag), None, None, None))
+        fields.byline.map(byline => Byline(byline, contributorTags))
```

Comment:
Minor style preference but how about this?

    val contributorApiTags = apiContent.tags.filter(_.`type` == TagType.Contributor)

and:

     fields.byline.map(Byline(_, contributorTags))